seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -50,7 +50,17 @@ public void setTableId(String tableId) {
         this.tableId = tableId;
     }
 
+    /**
+     * The method will be removed in the future, because the method name definition is not in
+     * accordance with the standard Pojo specification, the serialization framework cannot recognize
+     * it correctly.
```

Comment:
Please do not guarantee to developers that this class conforms to the POJO specification. Just tell devs use `setKind` is enough.

Reply:
done